lib/xsl/jats-to-html.xsl

```diff
@@ -521,7 +521,11 @@ xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xs="http://www.w3.org/2001/
         </xsl:template>
         <xsl:template match="//*[local-name()='math']">
             <span class="mathjax mml-math">
+<<<<<<< HEAD
                 <xsl:text disable-output-escaping="yes">&lt;math&gt;</xsl:text>
+=======
+                <xsl:text disable-output-escaping="yes">&lt;math xmlns:mml="http://www.w3.org/1998/Math/MathML"&gt;</xsl:text>
+>>>>>>> parent of a03930c... Fixes for remaining affiliation issues
```

Comment:
This is the reason why it is failing. This is evidence that there was an unresolved merge conflict in the jats-to-html.xsl file.